services/src/util/config.rs

```diff
@@ -108,6 +108,12 @@ pub struct Web {
     pub external_address: Option<url::Url>,
     pub backend: Backend,
     pub version_api: bool,
+    #[cfg(feature = "pro")]
+    #[serde(default)]
+    pub anonymous_access: bool,
+    #[cfg(feature = "pro")]
+    #[serde(default)]
+    pub user_registration: bool,
```

Comment:
This looks like polluting the open part with pro features.
Can't we make a config element solely for the pro context?